cli/azd/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 0.5.0-beta.3 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 0.5.0-beta.3 (2023-01-13)
 
 ### Bugs Fixed
 
```

Comment:
It is already at `0.5.0-beta.3` (https://github.com/Azure/azure-dev/blob/main/cli/version.txt#L1).  It was updated by the helpful bot earlier today: #1376.  I think I want to keep it as 0.5.0-beta.3 because that's the version we are releasing right?